.github/workflows/run_tests_pr.yaml

```
@@ -104,3 +136,5 @@ jobs:
           PR_TITLE_VAR: ${{ github.event.pull_request.title }}
           COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
           HEAD_COMMIT_MSG: ${{ github.event.head_commit.message }}
+          ENVIRONMENT: ${{ inputs.input_env || "staging" }}
```

actionlint (1.7.8)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)